### PR TITLE
from a plain javascript file to module js

### PR DESCRIPTION
**Pull request by aymanesarrar**

### CODE REVIEW

**Comment by devkennyy** [comment]
I can't get through the landing page and the theme popup doesn't show.

**Comment by aymanesarrar** [comment]
wait it still has some bugs

File: main.mjs [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import { Run_Stage_Required_Scripts } from "./src/scripts/game.mjs";
+import { Previous_Stage } from "./src/scripts/stages.mjs";
+import { Next_Stage } from "./src/scripts/stages.mjs";
+
+window.onload = Run_Stage_Required_Scripts();
+const start = document.getElementsByClassName("btn");
+start[0].addEventListener("click", () => {
+  Next_Stage("start");
+});
```
Comment (devkennyy) [inline main.mjs:9]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
});
```

File: main.mjs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+const stage4 = document.getElementById("stage_4-btn");
+const prev_stage = document.getElementById("prev_stage");
+
+stage1.addEventListener("click", () => {
+  Next_Stage(1);
+});
+stage2.addEventListener("click", () => {
+  Next_Stage(2);
+});
+stage3.addEventListener("click", () => {
```
Comment (devkennyy) [inline main.mjs:22]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
stage3.addEventListener("click", () => {
```

File: main.mjs [hunk 3]
```diff
@@ -0,0 +1,108 @@
+import { Run_Stage_Required_Scripts } from "./src/scripts/game.mjs";
+import { Previous_Stage } from "./src/scripts/stages.mjs";
+import { Next_Stage } from "./src/scripts/stages.mjs";
+
+window.onload = Run_Stage_Required_Scripts();
+const start = document.getElementsByClassName("btn");
+start[0].addEventListener("click", () => {
```
Comment (devkennyy) [inline main.mjs:7]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
start[0].addEventListener("click", () => {
```

File: main.mjs [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+const stage2 = document.getElementById("stage_2-btn");
+const stage3 = document.getElementById("stage_3-btn");
+const stage4 = document.getElementById("stage_4-btn");
+const prev_stage = document.getElementById("prev_stage");
+
+stage1.addEventListener("click", () => {
+  Next_Stage(1);
+});
+stage2.addEventListener("click", () => {
+  Next_Stage(2);
```
Comment (devkennyy) [inline main.mjs:20]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
  Next_Stage(2);
```

File: main.mjs [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+});
+stage3.addEventListener("click", () => {
+  Next_Stage(3);
+});
+stage4.addEventListener("click", () => {
+  Next_Stage(4);
```
Comment (devkennyy) [inline main.mjs:26]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
  Next_Stage(4);
```

File: main.mjs [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+});
+prev_stage.addEventListener("click", () => {
```
Comment (devkennyy) [inline main.mjs:28]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
prev_stage.addEventListener("click", () => {
```

File: main.mjs [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+start[0].addEventListener("click", () => {
+  Next_Stage("start");
+});
+const stage1 = document.getElementById("stage_1-btn");
+const stage2 = document.getElementById("stage_2-btn");
+const stage3 = document.getElementById("stage_3-btn");
+const stage4 = document.getElementById("stage_4-btn");
+const prev_stage = document.getElementById("prev_stage");
+
+stage1.addEventListener("click", () => {
```
Comment (devkennyy) [inline main.mjs:16]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
stage1.addEventListener("click", () => {
```

File: main.mjs [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+});
+stage4.addEventListener("click", () => {
+  Next_Stage(4);
+});
+prev_stage.addEventListener("click", () => {
+  Previous_Stage(1);
+});
+//Global vars
+export const totalStages = 4;
+export let stage = 1;
```
Comment (devkennyy) [inline main.mjs:33]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
export let stage = 1;
```

File: main.mjs [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+stage1.addEventListener("click", () => {
+  Next_Stage(1);
+});
+stage2.addEventListener("click", () => {
+  Next_Stage(2);
+});
+stage3.addEventListener("click", () => {
+  Next_Stage(3);
+});
+stage4.addEventListener("click", () => {
```
Comment (devkennyy) [inline main.mjs:25]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
stage4.addEventListener("click", () => {
```

File: main.mjs [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+  Next_Stage("start");
+});
+const stage1 = document.getElementById("stage_1-btn");
+const stage2 = document.getElementById("stage_2-btn");
+const stage3 = document.getElementById("stage_3-btn");
+const stage4 = document.getElementById("stage_4-btn");
+const prev_stage = document.getElementById("prev_stage");
+
+stage1.addEventListener("click", () => {
+  Next_Stage(1);
```
Comment (devkennyy) [inline main.mjs:17]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
  Next_Stage(1);
```

File: main.mjs [hunk 11]
```diff
@@ -0,0 +1,108 @@
+import { Run_Stage_Required_Scripts } from "./src/scripts/game.mjs";
+import { Previous_Stage } from "./src/scripts/stages.mjs";
+import { Next_Stage } from "./src/scripts/stages.mjs";
+
+window.onload = Run_Stage_Required_Scripts();
+const start = document.getElementsByClassName("btn");
+start[0].addEventListener("click", () => {
+  Next_Stage("start");
+});
+const stage1 = document.getElementById("stage_1-btn");
```
Comment (devkennyy) [inline main.mjs:10]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
const stage1 = document.getElementById("stage_1-btn");
```

File: main.mjs [hunk 12]
```diff
@@ -0,0 +1,108 @@
+import { Run_Stage_Required_Scripts } from "./src/scripts/game.mjs";
+import { Previous_Stage } from "./src/scripts/stages.mjs";
```
Comment (devkennyy) [inline main.mjs:2]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
import { Previous_Stage } from "./src/scripts/stages.mjs";
```

File: main.mjs [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+});
+prev_stage.addEventListener("click", () => {
+  Previous_Stage(1);
+});
+//Global vars
+export const totalStages = 4;
+export let stage = 1;
+
+//Stage one vars
+export var stageOne_ClickCount = 0;
```
Comment (devkennyy) [inline main.mjs:36]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
export var stageOne_ClickCount = 0;
```

File: main.mjs [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+stage3.addEventListener("click", () => {
+  Next_Stage(3);
+});
+stage4.addEventListener("click", () => {
+  Next_Stage(4);
+});
+prev_stage.addEventListener("click", () => {
+  Previous_Stage(1);
+});
+//Global vars
```
Comment (devkennyy) [inline main.mjs:31]: Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
//Global vars
```

**Pull request closed without merging**